pkg/kor/kor.go

```diff
@@ -56,17 +61,29 @@ func GetKubeClient(kubeconfig string) *kubernetes.Clientset {
 	return clientset
 }
 
-func SetNamespaceList(namespace string, kubeClient *kubernetes.Clientset) []string {
-	var namespaces []string
-	if namespace != "" {
-		namespaces = append(namespaces, namespace)
-	} else {
-		namespaceList, err := kubeClient.CoreV1().Namespaces().List(context.TODO(), metav1.ListOptions{})
-		if err != nil {
-			fmt.Fprintf(os.Stderr, "Failed to retrieve namespaces: %v\n", err)
-			os.Exit(1)
+func SetNamespaceList(lists IncludeExcludeLists, kubeClient *kubernetes.Clientset) []string {
+	namespaces := make([]string, 0)
+
+	includeNamespaceMap := make(map[string]struct{})
+	excludeNamespaceMap := make(map[string]struct{})
+	includeNamespaces := strings.Split(lists.IncludeListStr, ",")
+	excludeNamespaces := strings.Split(lists.ExcludeListStr, ",")
+	for _, ns := range excludeNamespaces {
```

Comment:
I think the exclude should only be taken into account in a case the include is not used(only when we are doing all namespaces).
We can skip looking at the exclude in this case and put a msg that it is ignored.

Reply:
I add some code. If --include-namespace is set, --exclude-namespaces will be ignored.